specs/capella/p2p-interface.md

```diff
@@ -59,11 +59,16 @@ See Capella [state transition document](./beacon-chain.md#beaconblockbody) for f
 
 This topic is used to propagate signed bls to execution change messages to be included in future blocks.
 
+Let `head_state` be a post state of the head of canonical chain.
```

Comment:
Just the chain or including any empty transitions to get to the current slot wrt wall time?

Reply:
Good point, I think we don't need the whole state in the definition of this change. I'll switch to `system_clock_epoch` (Thought this name for the variable would be less ambiguous than `current_epoch`)